examples/jaeger/README.md

```diff
@@ -0,0 +1,19 @@
+# Jaeger Example
```

Comment:
Add docker commands how to start `all-in-one` distribution

Reply:
Addressed in c15e5b1